src/KM_Controls.pas

```diff
@@ -350,6 +350,15 @@   TKMButtonFlat = class(TKMControl)
     procedure MouseUp(X,Y: Integer; Shift: TShiftState; Button: TMouseButton); override;
 
     procedure Paint; override;
+
+  end;
+
+
+  {FlatButton}
+  TKMButtonFlat = class(TKMButtonFlatCommon)

```

Comment:
TKMWareRow inherits same behaviour as TKMButtonFlat. Sound, highliting etc. So i extract common parent for both of them. The only difference - painting. 
Do you think its better just to copy paste ?